python/lsst/ts/ofc/bend_mode_to_force.py

```diff
@@ -94,7 +95,7 @@ def __init__(self, component, ofc_data):
             :, usecols
         ]
 
-    def force(self, dof):
+    def force(self, dof: np.ndarray[float]) -> np.ndarray:
```

Comment:
For the return values and others, if you use the `list`, `dict`, `np.ndarray`, or `tuple`, please try to be more specific to say `np.ndarray[float]`, `np.ndarray[int]`, etc. But if this would be too complex to affect the readability, you could ignore it. This comment applies to others. Thanks!